web/util.py

```diff
@@ -24,3 +25,19 @@ def flash_redirect(path: str, message: str | None = None, category="info"):
         flash(message, category)
 
     return redirect(path)
+
+
+def upload_file_to_printer(app, file, web_upload=False):
+    user_name = request.headers.get("User-Agent", "ankerctl").split(url_for('app_root'))[0]
+
+    with app.svc.borrow("filetransfer") as ft:
+        try:
+            ft.send_file(file, user_name)
+        except ConnectionError as err:
+            log.error(f"Connection error: {err}")
+            raise ConnectionError(err)
+        except Exception as err:
+            log.error(f"Unknown error: {err}")
+            raise Exception(err)
+
+    return {}
```

Comment:
We're not using the result, so we shouldn't have a `return` here.

Reply:
The return here is key in telling the function that called the upload that it is completed, otherwise it doesn't redirect. to tell you the upload was successful. I tested this many times with and without `return`, return was necessary. 

✅

Reply:
Sorry, my bad! I mixed up some stuff.

I meant to refer to `upload_file_to_printer()` (which *does* have an unnecessary return)